From ba38a6bdc88476a0df857eef28c6100dba79c221 Mon Sep 17 00:00:00 2001
From: Topi Reinio <topi.reinio@qt.io>
Date: Tue, 31 May 2022 09:48:58 +0200
Subject: [PATCH 173/181] Doc: Fix typo in the online documentation template

Pick-to: 5.15 6.2 6.3
Change-Id: I15e6db96722f5139f35dcffe70f796c4d7c17327
Reviewed-by: Venugopal Shivashankar <Venugopal.Shivashankar@qt.io>
(cherry picked from commit 11ccd2a68e041248da2695ff117cff24a9155dbe)
---
 doc/global/qt-module-defaults-online.qdocconf | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/doc/global/qt-module-defaults-online.qdocconf b/doc/global/qt-module-defaults-online.qdocconf
index a94195bd6c..b3a6df1191 100644
--- a/doc/global/qt-module-defaults-online.qdocconf
+++ b/doc/global/qt-module-defaults-online.qdocconf
@@ -12,7 +12,7 @@ HTML.footer = \
     "   <a href=\"http://www.gnu.org/licenses/fdl.html\">GNU Free Documentation" \
     "   License version 1.3</a> as published by the Free Software Foundation. " \
     "   Qt and respective logos are <a href=\"https://doc.qt.io/qt/trademarks.html\">" \
-    "   trademarks</a> of The Qt Company Ltd. in Finland and/or other countries\n"
+    "   trademarks</a> of The Qt Company Ltd. in Finland and/or other countries\n" \
     "   worldwide. All other trademarks are property of their respective owners. </p>\n"
 
 #include standard set of macros and C++ defines and ignores
-- 
2.40.0.rc0

